framework_tvb/tvb/interfaces/command/demos/project/project_export.py

```diff
@@ -57,10 +51,9 @@ def run_export(project_id, loose_irrelevant=False):
 
 
 if __name__ == '__main__':
+    from tvb.config.init.initializer import command_initializer
 
-    if len(argv) < 2:
-        print("You should specify a project ID to be exported!")
-
-    print("We will try to export project with ID: " + str(argv[1]))
+    command_initializer()
```

Comment:
Should we consider importing lab.py instead?
And promote usage of lab in all command examples as a good practice?